projects/core/src/cart/services/cart-config.service.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { TestBed } from '@angular/core/testing';
+import { CartConfig, CartConfigService } from '@spartacus/core';
+
+describe('CartConfigService', () => {
+  let mockCartConfig: CartConfig;
+  let service: CartConfigService;
+
+  beforeEach(() => {
+    // default spec cart config
+    mockCartConfig = {
+      cart: {
```

Comment:
What about reusing already defined defaultCartConfig?
(if we plan to keep it ;-) )

Reply:
Simplified ;)